orcestra/flightplan.py

```diff
@@ -485,6 +485,46 @@ def is_entry_through(self):
             return False
         return (self.enter + 180) % 360 - 180 > 0
 
+    def get_intersect(self, other: IntoCircle):
+        fwd, _, distance = geod.inv(
+            self.center.lon, self.center.lat, other.center.lon, other.center.lat
+        )
+        if distance > self.radius + other.radius:
+            raise ValueError(
+                "The two circles are too far apart. There is no intersection"
+            )
+        else:
+
+            def cost(fwd_az):
+                fwd_az = (fwd_az + 180) % 360 - 180
+                # print(fwd_az)
+                fwd_res = (
+                    geod.fwd(self.center.lon, self.center.lat, fwd_az, self.radius),
+                )
+                _, _, dist = geod.inv(
+                    fwd_res[0][0], fwd_res[0][1], other.center.lon, other.center.lat
+                )
```

Comment:
I'd suggest to destructure the result of `geod.fwd` as you do for `geod.inv` to improve readability:
```suggestion
                candidate_lon, candidate_lat, _ = (
                    geod.fwd(self.center.lon, self.center.lat, fwd_az, self.radius),
                )
                _, _, dist = geod.inv(
                    candidate_lon, candidate_lat, other.center.lon, other.center.lat
                )
```
The `candidate_lon`, `candidate_lat` result should be scalar. If not, then maybe `fwd_az ` isn't scalar (but maybe should?)

Reply:
yes so true. changed it